modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts

```diff
@@ -83,6 +84,11 @@ export class JournalEditArticlePage {
 		await this.page.locator('body').click();
 	}
 
+	async fillContent(content: string) {
+		await this.journalPage.webContentBodyTextBox.fill(content);
+		await this.journalPage.webContentBodyTextBox.press('Backspace');
```

Comment:
Hey @ambrinchaudhary, because if not the state of the ddm field does not change and it's translation status neither, so we need to add an interaction to it to update the status. The same was done here: https://github.com/liferay/liferay-portal/blob/e66d125cfceb1d0d651cffaf970b69c906fce536/modules/test/playwright/tests/journal-web/pages/JournalPage.ts#L58